logs-archives.go

```diff
@@ -0,0 +1,82 @@
+package scalingo
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"strconv"
+
+	errgo "gopkg.in/errgo.v1"
+)
+
+type LogsItem struct {
```

Comment:
Please these are not LogsItem, but LogsArchiveItem, be consistent with the naming, thanks.